planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/utils/utils.hpp

```diff
@@ -266,6 +266,14 @@ ExtendedPredictedObjects transform_to_extended_objects(
 double get_distance_to_next_regulatory_element(
   const CommonDataPtr & common_data_ptr, const bool ignore_crosswalk = false,
   const bool ignore_intersection = false);
+
+double get_min_dist_to_current_lanes_obj(
+  const CommonDataPtr & common_data_ptr, const FilteredByLanesExtendedObjects & filtered_objects,
+  const double dist_to_target_lane_start, const PathWithLaneId & path);
+
+bool has_blocking_target_object_for_stopping(
```

Comment:
Might be preferrable to have simpler name like `has_blocking_target_lane_object` and add docstring to explain the details?

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/9070/commits/6b6d64e595bdd56f242c7f477707d8ade7043606